extension/src/context.ts

```diff
@@ -29,7 +29,7 @@ export class ContextManager{
 	get ExtPath(): string { 
 		return this._ctx ? this._ctx.extensionPath : ""; 
 	}
-	get Data() { 
+	get Resources() { 
```

Comment:
Note, nothing in the Repo was using this, it was exposed for planned changes to AutoCompletion